docs: update v2.0.6 instruction (#33)
medibloc/panacea-mainnet: panacea-3/v2.0.6/upgrade.md

```

Then, stop your process.
```bash
pkill panacead

# or by other commands depending on your environment
```

After that, you should build a new binary as below. And, please replace the old `panacead` binary with the new one.

```bash
git clone https://github.com/medibloc/panacea-core
cd panacea-core
git checkout v2.0.6
make install

panacead version
# should print v2.0.6
```

Prior to the upgrade, validators are encouraged to take a full data snapshot. Snapshotting depends heavily on infrastructure, but generally this can be done by backing up the `~/.panacea` directory as below.<br>
**IMPORTANT**: It is critically important for validator operators to back-up the `~/.panacea/data/priv_validator_state.json` file after stopping the `panacead` process. This file is updated every block as your validator participates in consensus rounds. It is a critical file needed to prevent double-signing, in case the upgrade fails and the previous chain needs to be restarted.
```bash
cp -R ~/.panacea ~/panacea-3-v2.0.5-backup

# or, to the external storage
```

Then, you can start the `panacead` again.
```bash
panacead start

# or by other commands depending on your environment
```

**NOTE**:

> If you are using the [Cosmovisor](https://medibloc.gitbook.io/panacea-core/guide/cosmovisor) process manager, please build the new `panacead` binary manually and put that under the `$HOME/.panacea/cosmovisor/upgrades/v2.0.6/bin/` by following the [guide](https://medibloc.gitbook.io/panacea-core/guide/cosmovisor#cosmovisor-setup). The [auto-download](https://github.com/cosmos/cosmos-sdk/tree/main/cosmovisor#auto-download) is not supported yet because the appropriate version of the [libwasmvm.so](https://github.com/CosmWasm/wasmvm/blob/v0.14.0/api/libwasmvm.so) must be installed as well. Instead of installing the `libwasmvm.so` separately, I would recommend you build the `panacea-core` by the guide above. Then, the `libwasmvm.so` will be installed automatically.

Reviewer: We no longer use `wasm` in out upgraded binaries . So you can modify this part.
You could just as easily modify it to look like this
```suggestion
If you are using the [Cosmovisor](https://medibloc.gitbook.io/panacea-core/guide/cosmovisor) process manager, please build the new `panacead` binary manually and put that under the `$HOME/.panacea/cosmovisor/upgrades/v2.0.6/bin/` by following the [guide](https://medibloc.gitbook.io/panacea-core/guide/cosmovisor#cosmovisor-setup). We do not support automatic downloads for security reasons.
```

> **NOTE**: Since Cosmos SDK v0.43, a new configuration [grpc-web](https://github.com/cosmos/cosmos-sdk/blob/2582f0aab7b2cbf66ade066fe570a4622cf0b098/server/config/toml.go#L182) has been added to the `app.toml`. This configuration is enabled by default with a port 9091. If you don't want this feature, please disable it or change the port.

Reviewer: I think you can remove this part.

> **NOTE**: Since Cosmos SDK v0.43, a new configuration [grpc-web](https://github.com/cosmos/cosmos-sdk/blob/2582f0aab7b2cbf66ade066fe570a4622cf0b098/server/config/toml.go#L182) has been added to the `app.toml`. This configuration is enabled by default with a port 9091. If you don't want this feature, please disable it or change the port.

Reviewer: I think you can remove this part.

Reviewer: I'm talking about this part.
```
**NOTE**: Since Cosmos SDK v0.43, a new configuration [grpc-web](https://github.com/cosmos/cosmos-sdk/blob/2582f0aab7b2cbf66ade066fe570a4622cf0b098/server/config/toml.go#L182) has been added to the `app.toml`. This configuration is enabled by default with a port 9091. If you don't want this feature, please disable it or change the port.
```

Author: Fixed in[a58d1e6](https://github.com/medibloc/panacea-mainnet/pull/33/commits/a58d1e60372bce839fde4f34aa2b671bbcc75cb1).
